### DISCUSSION TITLE
Table Component?

### DISCUSSION THREAD

**Discussion by rcornwal** [mondaycom/vibe#1210] [General]
I was just wondering if there were any table components available in the Vibe system.

I do see virtual grid, and virtual list, but is there anything table specific?

**Comment by shlomitc** [comment]
Hey. A table component doesn't exist yet, but we do have future plans on adding such.
We will update once we create it

**Comment by shlomitc** [comment]
The first phase of our Table was just released:
https://style.monday.com/?path=/docs/data-display-table--overview

It's a relatively simple and straightforward data table that supports common UI, loading state, and scrolling capabilities out of the box.
I'd suggest using [`react-table`](https://tanstack.com/table/v8) or some other smart utilities to manage the table state.

Let us know if you find it useful
